src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/open.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends '@WebProfiler/Profiler/base.html.twig' %}
+
+{% block head %}
+    <style>
+        {{ include('@WebProfiler/Profiler/open.css.twig') }}
+    </style>
+{% endblock %}
+
+{% block body %}
+<div id="header">
+    <h1>{{ file }} <small>line {{ line }}</small></h1>
```

Comment:
would be nice to add a link to the doc so that people can learn about how to configure the IDE as target